README.md

```diff
@@ -44,6 +44,7 @@ Autocomplete speeds up the coding process by intuitively suggesting the most lik
 - Code Snippets
 - Copybook variable and paragraph names
 - Names of copybooks that are used in the program
+- CICS keywords inside `EXEC CICS` statements
```

Comment:
I think that specifying "inside EXEC CICS" is misleading - we don't check where the complete suggestions were requested from, so every time we return the CICS keywords together with COBOL ones.

Reply:
changed in latest commit